tlsh.go

```diff
@@ -285,11 +340,31 @@ func HashBytes(blob []byte) (hash string, err error) {
 
 //Hash calculates the TLSH for the input file
 func Hash(filename string) (hash string, err error) {
-	f, err := os.Open(filename)
+	hash, err = hashFile(filename)
+	if err != nil {
+		return "", err
+	}
+
+	return hash, nil
+}
+
+// Diff calculate distance between two files
+func Diff(filenameA, filenameB string) (int, error) {
```

Comment:
Is seems like `Diff()` should also exist on `Tlsh{}` and this package level one should just be a convenience method.